.travis.yml

```diff
@@ -5,7 +5,6 @@ node_js:
   - '7'
   - '6'
   - '5'
-  - '4'
```

Comment:
i think this should still be included, otherwise it's technically a breaking change (and if you're dropping 4, why not 5)

Reply:
I'd suggest using babel rather than allowing syntax to force version support; short of that, 5 is EOL so you might as well up it to 6 if you're dropping 4.

Reply:
well, that's fair, I'd rather have babel setup. Will do that.